README.md

```diff
@@ -74,17 +76,26 @@ Next, update your `AppDelegate` to include the following function to allow the r
 import OktaAuth
 
 func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
-  return OktaAuth.resume(url: url, options: options)
+  // oktaAuth - Configured OktaAppAuth instance used to start SignIn/SignOut flow. 
+  return oktaAuth.resume(url: url, options: options)
 }
 ```
 
 ## Usage Guide
 
-For an overview of this library's features and authentication flows, check out [our developer docs](https://developer.okta.com/code/ios).
+To start using our SDK you have to create a new object of  `OktaAppAuth`. You can instantiate  `OktaAppAuth` w/o parameters that means that SDK will use `Okta.plist` for configuration values. Alternatively you can create `OktaAppAuth` with custom configuration. For details about ways to configure `OktaAppAuth` see the [Configuration Reference](#configuration-reference)
+
+```swift
+import OktaAuth
 
-<!--
-TODO: Once the developer site provides code walkthroughs, update this with a bulleted list of possible flows.
--->
+// OktaAppAuth object with default configuration
+let defaultOktaAuth = OktaAppAuth()
+
+// OktaAppAuth object with custom configuration 
+let customConfig = OktaAuthConfig(...)
+let customOktaAuth = OktaAppAuth(configuration: customConfig)
```

Comment:
nit: Use comments to describe what is happening, instead of variables:

```swift
// Use the default Okta.plist configuration
let oktaAppAuth = OktaAppAuth()

// Use configuration from another resource
let config = OktaAuthConfig(/* plist */)
let config = OktaAuthConfig(/* dictionary */)

// Instantiate OktaAuth with custom configuration object
let oktaAuth = OktaAppAuth(configuration: config)
```

Reply:
Done.